recipes/pyro5/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "5.5.1" %}
+
+package:
+  name: pyro5
+  version: {{ version }}
+
+source:
+  fn: Pyro5-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/P/Pyro5/Pyro5-{{ version }}.tar.gz
+  sha256: fe7998b0bfab7e6f1bf02fa7d69e4b4c53305da745bae07d604eeea139316828
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - pyro5-ns = Pyro5.nameserver:main
+    - pyro5-nsc = Pyro5.nsc:main
+    - pyro5-echoserver = Pyro5.utils.echoserver:main
+    - pyro5-check-config = Pyro5.configure:dump
+    - pyro5-httpgateway = Pyro5.utils.httpgateway:main
+
+
+requirements:
+  build:
```

Comment:
Just ``host`` should be enough, could you please remove the ``build``?

Reply:
ok